internal/quickstart/toggle_flag.go

```diff
@@ -53,9 +54,20 @@ func (m toggleFlagModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		case key.Matches(msg, pressableKeys.Tab):
 			m.flagWasEnabled = true
 			m.enabled = !m.enabled
+			m.err = nil
 			return m, toggleFlag(m.client, m.accessToken, m.baseUri, m.flagKey, m.enabled)
 		}
 	case errMsg:
+		msgRequestErr, err := newMsgRequestError(msg.err.Error())
```

Comment:
We handle the error here since we're mapping the API error response to a model-specific response.